staging_vespalib/src/vespa/vespalib/util/shutdownguard.cpp

```diff
@@ -1,11 +1,23 @@
 // Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 #include "shutdownguard.h"
+#include <unistd.h>
+#include <sys/time.h>
 
 #include <vespa/log/log.h>
 LOG_SETUP(".vespalib.shutdownguard");
 
 namespace vespalib {
 
+namespace {
+enum { STACK_SIZE = (1u << 16) };
+
+static uint64_t getTimeInMillis() {
```

Comment:
Intentionally inherited `static`-ness from when it was inside class `ShutdownGuard`?